data_taking_certification/views.py

```diff
@@ -3,58 +3,45 @@
 from data_taking_objects.models import Run, Lumisection
 from data_taking_certification.models import RunCertification, LumisectionCertification
 
-# from histograms.models import RunHistogram
-
-import pandas as pd
-
 
 # Create your views here.
 def run_certification_view(request):
 
     error_message = None
-    df = None
-
-    # TODO The following lines should be done on the DB side
-    certification_df = pd.DataFrame(RunCertification.objects.all().values())
-    runs_df = pd.DataFrame(Run.objects.all().values())
-
-    if certification_df.shape[0] > 0:
-        df = runs_df.merge(certification_df, left_on="id", right_on="run_id")[
-            [
-                "run_number",
-                "rr_frac_pixel_good",
-                "rr_frac_strip_good",
-                "rr_frac_tracking_good",
-                "rr_frac_ecal_good",
-            ]
-        ]
-        print(df.head())
+
+    run_certifications = RunCertification.objects.all()
+    n_certifications = len(run_certifications)
```

Comment:
Just a note: it's generally recommended to use [`count()`](https://docs.djangoproject.com/en/4.0/ref/models/querysets/#django.db.models.query.QuerySet.count) instead of `len`, since the latter has to evaluate the query, so it's slower.

Reply:
Done.